packages/cf-component-copyable-textarea/package.json

```diff
@@ -15,6 +15,7 @@
   },
   "devDependencies": {
     "assert-equal-jsx": "^1.0.3",
+    "cf-util-text": "^2.0.0",
```

Comment:
https://github.com/cloudflare/cf-ui/blob/master/packages/cf-component-copyable-textarea/example/basic/component.js#L5

It was breaking the example dev builds for me